spec.md

```diff
@@ -1849,6 +1849,24 @@ message NodePublishVolumeRequest {
   // MUST match the attributes of the Volume identified by
   // `volume_id`.
   map<string, string> volume_attributes = 8;
+
+  // Information about workload that wants to use the volume.
+  // This is an OPTIONAL field. The Plugin MAY choose to ignore
+  // this field if it does not need any workload related information
+  // to fullfill the request.
+  Workload workload = 9;
+}
+
+// Parameters of a CO workload that lead to CSI request.
+message Workload {
+    // CO name. This name defines content of parameters map. Each CO
```

Comment:
nits: fix indentation. the indentation should be 2 here.

Reply:
fixed